test/gui/shared/steps/steps.py

```diff
@@ -398,13 +398,47 @@ def createFolder(context, foldername, username=None):
     path = join(syncPath, foldername)
     os.makedirs(path)
 
+ 
+@When('user "|any|" creates "|any|" files each of size "|any|" bytes inside the folder "|any|"')
+def step(context, username, filenumber, filesize, foldername):
+    syncPath = context.userData['currentUserSyncPath']
+    path = join(syncPath, foldername)
+    filesize = builtins.int(filesize)
+    snooze(5)
```

Comment:
Please include appropriate reason for the static wait.

Reply:
Syncing all the created files at once seems to have some problem on syncing, as a result not all created files sync successfully. So once a file is synced the process will have to wait unless the next file is created. Also, it takes at least 5 seconds for a file to begin the sync process. So here we added a snooze of 5 seconds.

Also, the step ```When the user waits for the files to sync``` is not working properly as expected. And probably this PR -> https://github.com/owncloud/client/pull/9442 may solve this issue. So we can’t remove snooze(5) unless the issue is fixed.